tools/cleanup-precomps.raku

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env raku
+
+my str $compilation-id = Compiler.id;
+
+#| Remove old precomp files from the repository chain
+unit sub MAIN();
+
+sub unlink-recursively(IO::Path:D $io, :$keep --> Int:D) {
+    my int $bytes;
+    for $io.dir {
+        if .d {
+            $bytes += unlink-recursively($_)
+              if .basename ne $compilation-id;
+        }
+        else {
+            $bytes += .s;
+            .unlink;
+        }
+    }
+
+    $io.rmdir unless $keep;
+    $bytes
+}
+
+my int $repositories;
+my int $total;
+my @repos := $*REPO.repo-chain;
+for $*REPO.repo-chain.map({ $_ with .?prefix }) {
```

Comment:
I think this script is heavily coupled to one particular workflow in a way that can lead to confusion. For instance: sould I, someone with multiple raku installations, expect such a script to delete the precomp folders for all but one compiler in `~/.raku`? Or to try to do anything for non-`CURI` repositories that happen to also use a `prefix`?

Reply:
It may lead to confusion in such cases.

Perhaps I should just put this script into module space.

Reply:
Removed with https://github.com/rakudo/rakudo/commit/257e06ab5f